deploy.yaml

```diff
@@ -0,0 +1,27 @@
+#To use this you will need buildcomponents.yaml in your top level ci file. 
+publish:
+  image: briangweber/docker-node:carbon
+  stage: publish
+  script:
+    - grunt bundle --packageVersion=$VERSION --packageName=$CI_PROJECT_NAME
+  only:
+    - master
+  except:
+    - schedules
+
+deploy-int:
+  extends: .deploy
+  stage: deploy-int
+  variables:
+    DEPLOYENV: int
```

Comment:
Instead of setting this, by setting `deploy-int.environment.name`, gitlab ci sets the `CI_ENVIRONMENT_NAME` environment variable to "integration". Two birds with one stone if you use that. Would probably want to change the environment name to "int" though.